src/phoenix/server/api/types/AddSpansToDatasetPayload.py

```diff
@@ -1,8 +0,0 @@
-import strawberry
```

Comment:
@axiomofjoy nixed for now as they all are the same - happy to restore but just keeps the schema a bit tighter. WYT

Reply:
Not sure I have a strong opinion here. What does GitHub do?

Reply:
I think for now let's keep it shorter.

Reply:
Sounds good. For reference, looks like GitHub creates output types for each mutation.

![Screenshot 2024-05-22 at 10 45 30 AM](https://github.com/Arize-ai/phoenix/assets/15664869/a27e769a-8ba4-41f9-a52a-cce74abe520d)

![Screenshot 2024-05-22 at 10 45 42 AM](https://github.com/Arize-ai/phoenix/assets/15664869/78330de2-6eed-4094-aa6e-76c68cf93e91)